CMakeLists.txt

```diff
@@ -2,8 +2,6 @@
 # Licensed under the Apache 2.0 License.
 cmake_minimum_required(VERSION 3.11)
 
-add_definitions(-DOE_API_VERSION=2)
```

Comment:
Do we not need this anymore?

Reply:
No, not since 0.7, I'm cleaning up cmake as a way to re-trigger the CI, I'll pull the changes separately.